core/src/main/java/com/graphhopper/routing/weighting/NoTurnCostProvider.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.graphhopper.routing.weighting;
+
+public class NoTurnCostProvider implements TurnCostProvider {
```

Comment:
Maybe we should avoid instantiating this provider and create a singleton from it? Or even remove this class and create a variable "NO_TURN_COST" in TurnCostProvider?

Reply:
Ok, will do.